wondrous-app/components/Common/TaskSubmission/styles.tsx

```diff
@@ -795,3 +791,26 @@ export const SubmissionDescription = styled.div`
     `}
   }
 `;
+
+export const GiveKudosButton = styled(ButtonBase)`
```

Comment:
can you use button from `components/Button`?

Reply:
Using `components/Button` now but this needs some customization. Let me know your thoughts.